bindings/cpu_profiler.cc

```diff
@@ -267,7 +269,7 @@ static void StopProfiling(const v8::FunctionCallbackInfo<v8::Value>& args) {
   std::string app_root_directory = args[2].IsEmpty() ? std::string() : std::string(*Nan::Utf8String(args[2]));
   uint32_t thread_id = Nan::To<uint32_t>(args[1]).FromJust();
 
-  v8::Local<v8::Value> js_profile = CreateProfile(profile, thread_id, app_root_directory);
+  v8::Local<v8::Value> js_profile = CreateProfile(*profile, thread_id, app_root_directory);
```

Comment:
You shouldn't need to dereference `*profile` if it's being passed as a reference